### PR TITLE
[bitnami/minio] Update docs for distributed mode

### PR DESCRIPTION
**Pull request by jani888**
I've added extra documentation about the defaultBuckets property, and that it should be disabled when running in distributed mode

### CODE REVIEW

File: bitnami/minio/README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ The command removes all the Kubernetes components associated with the chart and
...
 | `auth.useCredentialsFiles` | Mount credentials as a files instead of using an environment variable                                                                                                                                     | `false`                        |
 | `auth.forceNewKeys`        | Force root credentials (user and password) to be reconfigured every time they change in the secrets                                                                                                       | `false`                        |
-| `defaultBuckets`           | Comma, semi-colon or space separated list of buckets to create at initialization (only in standalone mode)                                                                                                | `""`                           |
+| `defaultBuckets`           | Comma, semi-colon or space separated list of buckets to create at initialization (only in standalone mode). Do not set this value in distributed mode, as it will cause MinIO&reg; to crash.                                                                                                | `""`                           |
```
Comment (carrodher) [inline bitnami/minio/README.md:98]: Please note this table is automatically updated using the metadata from the values.yaml, see https://github.com/bitnami/charts/blob/main/bitnami/minio/values.yaml#L119.

You should perform the change there and the table will be automatically updated by https://github.com/bitnami/readme-generator-for-helm

Comment (jani888) [reply bitnami/minio/README.md]: Okay, I've fixed it

**Comment by carrodher** [comment]
Thanks for the fix. One last note, it is needed to bump the chart version present in the _Chart.yaml_

**Comment by juan131** [comment]
@jani888 did you see last comment from @carrodher? Please ensure you merge in your branch the latest changes from `main` and then you bump the chart version present in the **Chart.yaml**.

**Comment by jani888** [comment]
@juan131 sorry, I was very busy with work. I just had some time to take a look at this. I've merged the main branch, and bumped the version. Is there anything else I should do?

**Comment by juan131** [comment]
Hi @jani888 

It seems you messed up sth while merging master in your branch, see:

Screenshot 2024-01-22 at 09 55 06

Could you please solve this?

**Pull request closed without merging**